constructorio_python/constructorio.py

```diff
@@ -0,0 +1,56 @@
+'''ConstructorIO Python Package'''
+
+__version__ = 1.0
+
+from constructorio_python.helpers.exception import ConstructorException
+from constructorio_python.modules.autocomplete import Autocomplete
+from constructorio_python.modules.browse import Browse
+from constructorio_python.modules.recommendations import Recommendations
+from constructorio_python.modules.search import Search
+
+
+class ConstructorIO:
+    '''
+        ConstructorIO Python Client
+
+        :param str api_key: Constructor.io API key
+        :param str api_token: Constructor.io API token
+        :param str security_token: Constructor security token
+        :param str service_url: API URL endpoint
+
+        :return: class
+    '''
+
+    def __init__(self, options) -> None:
```

Comment:
should `options` be `**kwargs` instead?

Reply:
We take a big dictionary instead of individual arguments. We kept it this way to follow the same function signature of other SDKs we have